shell/content/docs/en-us/whats-new.md

```diff
@@ -2,6 +2,33 @@
 title: What's New in 2.6
 ---
 
+
+## New in 2.6.7
+
+### Amazon Billing in the Marketplace
+
+- Now you can purchase support for Rancher through the AWS marketplace 
+
+### Ability to configure Azure AD with MSAL
+
+- [need description]
+
+### Kubernetes 1.24
+
+- Kubernetes 1.24 is now available as a supported Kubernetes version option when provisioning RKE1 clusters as well as upgrading imported RKE2/k3s clusters.
+
+### RKE2 Encryption Key Rotation
+
+- [need description]
```

Comment:
```suggestion
- RKE2 encrypts secret data at rest by default. You can now rotate the key used for this encryption for Rancher-provisioned RKE2 clusters with supported k8s versions.
```

@jakefhyde @Oats87 , please correct / add as necessary

Reply:
It might be worth calling out that this feature is also available for k3s as well (although encryption not enabled by default) for clarity. Also I know we are release noting https://github.com/rancher/rke2/issues/3186, but should we also call it out here as well?

Reply:
@jakefhyde , thank you!

I believe we generally want short statements on UI splash screen highlighting the features. So I don't believe we want to list all known issues and such here. But I'll let @gaktive decide.
Also, the actual issue that we will be release noting is this one: https://github.com/rancher/rancher/issues/38283